src/index.ts

```diff
@@ -148,6 +148,19 @@ export class Collection<K, V> extends Map<K, V> {
 		return arr.slice(-amount);
 	}
 
+	/**
+	 * Identical to [Array.at()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at).
+	 * Returns the item at a given index, allowing for positive and negative integers.
+	 * Negative integers count back from the last item in the array.
```

Comment:
Isn't here better to use `collection` as we're not really working with arrays (although it's an array method)?
```suggestion
	 * Negative integers count back from the last item in the collection.
```

Reply:
Eh sure
I think I did that because we transform the collection into an array and I did the JSDoc afterwards, so got a little confusing